requirements.txt

```diff
@@ -2,5 +2,7 @@ Django==1.10
 django-cors-middleware==1.2.0
 django-extensions==1.7.1
 djangorestframework==3.4.4
+psycopg2==2.7.7
+psycopg2-binary==2.7.7
```

Comment:
This is trivial but pyscopg2-binary makes psycopg2 obsolete so there should not be a need for use of both at the same time.